packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -141,29 +141,19 @@ export const useArbTokenBridge = (
   const [pendingWithdrawalsMap, setPendingWithdrawalMap] =
     useState<PendingWithdrawalsMap>({})
 
+  const l1NetworkID = useMemo(() => String(l1.network.id), [l1.network])
+  const l2NetworkID = useMemo(() => String(l2.network.id), [l2.network])
+
   // once the l1/l2/account changes, we need to revalidate the withdrawal list in the store
   // this prevents previous account/chains' transactions to show up in the current account
   // also makes sure the state of app doesn't get incrementally bloated with all accounts' txns loaded up till date
   useEffect(() => {
-    if (!connector) {
+    if (!l1NetworkID || !l2NetworkID || !walletAddress) {
```

Comment:
Why is a check for `l1NetworkId` and `l2NetworkID` necessary? Can they be null/undefined?

Reply:
Upon further checking, no it was not necessary as `arbtokenbridge` can only be instantiated with a valid network connection. So network ID will always be present, as indicated by our type checks as well.